app/src/main/java/com/android/sample/ui/components/Textfields.kt

```diff
@@ -55,14 +69,25 @@ fun EmailTextField(
     onEmailChange: (String) -> Unit,
     emailError: String?,
 ) {
-  OutlinedTextField(
-      value = email,
-      onValueChange = onEmailChange,
-      label = { Text("Email") },
-      isError = emailError != null,
-      modifier = Modifier.fillMaxWidth(0.8f).testTag("EmailTextField"))
-  emailError?.let {
-    Text(text = it, color = Color.Red, fontSize = 12.sp, modifier = Modifier.padding(start = 16.dp))
+  Column(modifier = Modifier.fillMaxWidth(0.8f)) {
```

Comment:
Same here : the addition of a column allows you to put the error part, not using a lazy column should not be bad in this case as there is only one field to show in this composable.